Controller/ResetPasswordController.php

```diff
@@ -181,6 +181,7 @@ public function reset(Request $request, string $token)
 
             $user->setPassword($password);
             $user->setConfirmationToken(null);
+            $user->setEnabled(true);
```

Comment:
@rvmourik is dit gewenst? We zouden ook kunnen stellen een gedeblokkeerde ook niet een wachtwoord reset zou mogen doen... (ook niet vanuit een admin)

Reply:
@rvmourik ☝️ 😄

Reply:
Heb overleg gehad met @mwansinck hij moet er idd uit.